.github/actions/deploy/action.yml

```diff
@@ -0,0 +1,11 @@
+name: Deploy
```

Comment:
The name seems to be quite generic to me, we could call this action `package-deploy/action.yml` for example, what do you think?

Reply:
Your right, i'll update